pkg/handlers/index/list.go

```diff
@@ -16,13 +16,12 @@
 package index
 
 import (
-	"net/http"
-	"sort"
-
 	"github.com/gin-gonic/gin"
-
 	"github.com/zinclabs/zinc/pkg/core"
 	"github.com/zinclabs/zinc/pkg/meta"
+	"net/http"
+	"sort"
+	"strings"
```

Comment:
The import should be format, it will separate to three part:

1. standard library
2. third part library 
3. project own namespace

you can use go-import to format it, after format it will be like this:

```
import (
    "net/http"
    "sort"
    "strings"

    "github.com/gin-gonic/gin"

    "github.com/zinclabs/zinc/pkg/core"
    "github.com/zinclabs/zinc/pkg/meta"
)
```